hips/tiles/tile_meta.py

```diff
@@ -19,12 +27,22 @@ class HipsTileMeta:
         File format
     tile_width : `int`
         Tile width (in pixels)
+
+    Examples
+    --------
+    >>> from hips.tiles import HipsTileMeta
+    >>> tile_meta = HipsTileMeta(order=3, ipix=450, file_format='fits', frame='galactic', tile_width=512)
+    >>> tile_meta.skycoord_corners
+    <SkyCoord (Galactic): (l, b) in deg
+    [( 264.375, -24.62431835), ( 258.75 , -30.        ),
+     ( 264.375, -35.68533471), ( 270.   , -30.        )]>
     """
 
-    def __init__(self, order: int, ipix: int, file_format: str, tile_width: int = 512) -> None:
+    def __init__(self, order: int, ipix: int, file_format: str, frame='galactic', tile_width: int = 512) -> None:
```

Comment:
You need to add `frame` to the `Parameters` section in the docstring and document what the valid values are.
Maybe like this?
```
frame : {'icrs', 'galactic'}
    Sky coordinate frame
```